CMakeLists.txt

```diff
@@ -374,6 +374,7 @@ set(SRC_FILES
   src/include/proxy.h
   src/include/rccl_bfloat16.h
   src/include/rccl_vars.h
+  src/include/rccl_bfloat8.h
```

Comment:
One minor comment: shall we change to rccl_float8.h? Because it contains float8(e4m3) and bfloat8(e5m2). It also better aligns with RCCL_FLOAT8 macro naming.

Reply:
update according to cr comments